e2e.md

```diff
@@ -0,0 +1,63 @@
+**Running End-to-end Tests on Remote Clusters**
+
+This article outlines steps to run e2e tests on remote clusters for controllers created using `kubebuilder`. For example, after developing a database controller, the developer may want to run some e2e tests on a GKE cluster to verify the controller is working as expected. Currently, `kubebuilder` does not provide a template for running the e2e tests. This article serves to address this deficit.
+
+The steps are as follow:
+1.  Create a test file named `<some-file-name>_test.go` populated with template below (referring [this](https://github.com/foxish/application/blob/master/e2e/main_test.go)):
```

Comment:
Thank you for adding this file. 
Recently #229 is merged. Now there is a e2e test example inside kubebuilder you can refer
https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/e2e.go

Reply:
Should I just close this PR? Or should I change the PR to be based on the recently merged `e2e.go` so that we have one `e2e.md`?

Reply:
This is different from the PR that got merged. PR #229 is about testing Kubebuilder end to end while this PR is about documenting steps for writing an e2e for the controller that you generated using KB. So don't close this PR.